service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -194,6 +194,7 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 						},
 					},
 				},
+				EnableFeatures:            []string{"remote-write-receiver"},
 				EnableRemoteWriteReceiver: true,
```

Comment:
You need to remove this one also EnableRemoteWriteReceiver

Reply:
Why ? `EnableRemoteWriteReceiver` exists in 0.62.0